ui/imports/utils/Utils.qml

```diff
@@ -346,6 +353,31 @@ QtObject {
         }
     }
 
+    // WARN: It is not used!! TO BE REMOVE??
+    function formatAgeFromTime(timestamp, epoch) {
```

Comment:
@noeliaSD don't we need to display "now" in case of messages received in the last minute (it's worth checking the requirements)? Feel free to remove unused code.

Reply:
By searching the code I cannot see any call to these methods but maybe they can be useful in some point, that's why I cannot remove them!